cmd/gateway-unsupported.go

```diff
@@ -240,3 +240,8 @@ func (a GatewayUnsupported) IsCompressionSupported() bool {
 func (a GatewayUnsupported) IsReady(_ context.Context) bool {
 	return false
 }
+
+// IsReady - No Op.
+func (a GatewayUnsupported) IsClusterReady(_ context.Context) bool {
+	return false
```

Comment:
For gateways can we leverage the `IsBackendOnline` method used in readiness checkes?

Reply:
This is not meant for gateway's @nitisht and not meaningful.

Reply:
cool